_tour/implicit-conversions.md

```diff
@@ -58,3 +45,17 @@ To turn off the warnings take either of these actions:
 * Invoke the compiler with `-language:implicitConversions`
 
 No warning is emitted when the conversion is applied by the compiler.
+{% endtab %}
+{% tab 'Scala 3' %}
+Because implicit conversions can have pitfalls if used indiscriminately the compiler warns in two situations:
+- when compiling the implicit conversion definition (for Scala 2 style conversions).
```

Comment:
If the Scala 3 compiler does not warn on `given Conversion` instances, then I would make that more explicit:

```suggestion
- when compiling a Scala 2 style implicit conversion definition.
```